backend/de.metas.salescandidate.base/de.metas.ordercandidate_GenerateModels-table.run.xml

```diff
@@ -2,10 +2,20 @@
   <configuration default="false" name="de.metas.ordercandidate_GenerateModels-table" type="Application" factoryName="Application" folderName="generate models">
     <option name="INCLUDE_PROVIDED_SCOPE" value="true" />
     <option name="MAIN_CLASS_NAME" value="org.adempiere.util.GenerateModel" />
-    <module name="de.metas.salescandidate.base" />
-    <option name="PROGRAM_PARAMETERS" value="src/main/java-gen - '%' C_OLCand" />
+    <module name="de.metas.contracts" />
```

Comment:
pls revert this file....

Reply:
reverted. I've asked @metas-ts to chime in if he feels that's by design.

Reply:
sorry, that was an accident on my part. thx for fixing